dom.bs

```diff
@@ -6738,6 +6741,11 @@ invoked, must run these steps:
  <li><p>Set <var>shadow</var>'s <a for=ShadowRoot>delegates focus</a> to <var>init</var>'s
  {{ShadowRootInit/delegatesFocus}}.
 
+ <li><p>If <a>this</a> is <a for=Element>custom</a>, and if <a>this</a>'s <a for=Element>custom
+ element state</a> is not "precustomized" or "custom", set <var>shadow</var>'s
```

Comment:
If it's custom, why would it not be "custom"? Isn't that the definition of custom?

Reply:
You're right, I should re-use the definition of "custom" here. To do that, I added "precustomized" to the definition of "custom element" in the other PR, [here](https://github.com/whatwg/dom/pull/894/commits/cf4f0dbe73ab4a9a6e5c83a5a8b0ec9aebff5dc3). And then I changed this to just "if this is custom".